framework/update/helper/updatetask_win.cpp

```
@@ -0,0 +1,1387 @@
+        principal->put_UserId(userId);
+        ::SysFreeString(userId);
+
+        principal->put_LogonType(TASK_LOGON_SERVICE_ACCOUNT);
+        principal->put_RunLevel(TASK_RUNLEVEL_HIGHEST);
+        principal->Release();
+    }
+
+    //! NOTE: We are that helper, running from the installation the task is being
+    //! registered for - no need to guess where it was put.
+    const std::wstring helperPath = modulePath();
+    if (helperPath.empty()) {
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)